code/game/objects/effects/anomalies.dm

```diff
@@ -80,6 +85,16 @@
 	if(I.tool_behaviour == TOOL_ANALYZER)
 		to_chat(user, "<span class='notice'>Analyzing... [src]'s unstable field is fluctuating along frequency [format_frequency(aSignal.frequency)], code [aSignal.code].</span>")
 
+///freeze the anomaly and dont let it move or do weird anomaly stuff
+/obj/effect/anomaly/proc/Suspend()
+	suspended = TRUE
+	countdown.stop()
+
+///unfreeze the anomaly and LET IT RIP
+/obj/effect/anomaly/proc/Unsuspend()
+	suspended = FALSE
+	countdown.start()
+
```

Comment:
Could you make the anomaly stop processing on `Suspend()` and restart on `Unsuspend()`?